script/main.js

```diff
@@ -0,0 +1,44 @@
+(function(){
+    function getRandomInt (min, max) {
+        min = Math.floor(min);
+        max = Math.ceil(max);
+        return Math.floor(Math.random() * (max - min + 1)) + min;
+    }
+
+    let firstDice={
+        number:getRandomInt(1, 6),
+        roll: () => {
+            this.number = getRandomInt(1, 6);
+            return this.number;
```

Comment:
Well done, it is considered good practice to return a value even from object methods which mutate the object.